xep-0353.xml

```diff
@@ -262,21 +234,179 @@
+    In this case (e.g. no party answered the &lt;propose/&gt; message yet) the lower of the two session IDs MUST overrule the other action, where by "lower" is meant the session ID that is sorted first using "i;octet" collation as specified in Section 9.3 of &rfc4790; (in the unlikely event that the random session IDs are the same, the action sent by the lower of the JabberIDs MUST overrule the other action). The party that receives the &lt;propose/&gt; action with the lower of the two session IDs MUST respond with an &lt;accept/&gt; or &lt;reject/&gt; mesage like it would normally do for a &lt;propose/&gt; message, and the party that receives the &lt;propose/&gt; action with the higher of the two session IDs MUST return a &lt;reject/&gt; message to the other party with a &lt;tie-break/&gt; child element alongside of a &lt;reason/&gt; element carrying the condition &lt;expired/&gt;.
+      <example caption="Tie break in propose state"><![CDATA[
+<!-- lower session ID -->
+<message from='romeo@montague.example/orchard'
+         to='juliet@capulet.example'
+         type='chat'>
+  <propose xmlns='urn:xmpp:jingle-message:1' id='a73sjjvkla37jfea'>
+    <description xmlns='urn:xmpp:jingle:apps:rtp:1' media='audio'/>
+  </propose>
+  <store xmlns="urn:xmpp:hints"/>
+</message>
+
+<!-- higher session ID -->
+<message from='juliet@capulet.example/phone'
+         to='romeo@montague.example'
+         type='chat'>
+  <propose xmlns='urn:xmpp:jingle-message:1' id='b73sjjvkla37jfea'>
+    <description xmlns='urn:xmpp:jingle:apps:rtp:1' media='audio'/>
+  </propose>
+  <store xmlns="urn:xmpp:hints"/>
+</message>
+
+<!-- Romeo received the higher ID and rejects the call -->
+<message from='romeo@montague.example/orchard'
+         to='juliet@capulet.example'
+         type='chat'>
+  <reject xmlns='urn:xmpp:jingle-message:1' id='b73sjjvkla37jfea'>
+    <reason xmlns="urn:xmpp:jingle:1">
+      <expired/>
+      <text>Tie-Break</text>
+    </reason>
+    <tie-break/>
+  </reject>
+  <store xmlns="urn:xmpp:hints"/>
+</message>
+
+<!-- Juliet received the lower ID and accepts the call -->
+<message from='juliet@capulet.example/phone'
+         to='romeo@montague.example'
+         type='chat'>
+  <accept xmlns='urn:xmpp:jingle-message:1' id='a73sjjvkla37jfea'/>
+  <store xmlns="urn:xmpp:hints"/>
+</message>
+]]></example>
+  </section2>
+  <section2 topic='Existing session' anchor='tie-break-2'>
+    <p>If (from the perspective of the responder of the new session) there is already a session to the bare-jid of the initiator active (e.g. call already accepted but no &lt;finish/&gt; element received by the responder so far), the old session MUST be deemed orphan and terminated by the responder of the new session in favor of the new one. The responder MUST transparently accept the new session and finish the old one, because it can be assumed that this new session is a transparent continuation of the old one.</p>
```

Comment:
```suggestion
    <p>If (from the perspective of the responder of the new session) there is already a session to the bare-jid of the initiator active (e.g. call already accepted but no &lt;finish/&gt; element received by the responder so far), the old session MUST be deemed an orphan and terminated by the responder of the new session in favor of the new one. The responder MUST transparently accept the new session and finish the old one, because it can be assumed that this new session is a transparent continuation of the old one.</p>
```